server/TracySourceView.cpp

```diff
@@ -3843,69 +3843,48 @@ void SourceView::RenderAsmLine( AsmLine& line, const AddrStat& ipcnt, const Addr
     memcpy( buf, line.mnemonic.c_str(), msz );
     memset( buf+msz, ' ', m_maxMnemonicLen-msz );
     bool hasJump = false;
+    int jumpLabel = 0;
     if( line.jumpAddr != 0 )
     {
         auto lit = m_locMap.find( line.jumpAddr );
         if( lit != m_locMap.end() )
         {
+            jumpLabel = lit->second;
             char tmp[64];
-            sprintf( tmp, ".L%" PRIu32, lit->second );
-            strcpy( buf+m_maxMnemonicLen, tmp );
```

Comment:
I have asked you before to use the existing code style. This change does nothing *but* changes the style to be non-conforming.

Reply:
Can do; Is there a reason there is no clang-format file? These are artifacts of IDE fixups when copy pasting code around - very easy on my end to miss

Reply:
> Is there a reason there is no clang-format file?

Because someone would have to make it.

> These are artifacts of IDE fixups

Use different IDE then.

> very easy on my end to miss

I would expect people to actually look at the changes they propose (i.e. by running a `git diff` before commiting changes). Otherwise it may look like they do not respect my time.